src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -53,12 +76,34 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
           class="markdown ${classMap({
             "no-header": !this._config.title,
           })}"
-          .content="${this._config.content}"
+          .content="${this._content}"
         ></ha-markdown>
       </ha-card>
     `;
   }
 
+  private async _connect() {
+    if (!this._unsubRenderTemplate && this._hass && this._config) {
+      this._unsubRenderTemplate = await subscribeRenderTemplate(
```

Comment:
Please run this inside a `try…catch` and inside the catch, assign `this._content = this._config.content`. That way this card can run on older versions of Home Assistant. This is something that can happen when using with Home Assistant Cast.